library/firewall_lib.py

```diff
@@ -222,6 +265,43 @@ def parse_port(module, item):
     return (_port, _protocol)
 
 
+ipv4_charset = ".1234567890/"
+ipv6_charset = ":1234567890abcdef/"
+
+
+def parse_destination_address(module, item):
```

Comment:
You might also consider using the python ipaddress library https://docs.python.org/3/library/ipaddress.html

Reply:
Do we need to worry about supporting python 2.7? It looks like the ipaddress library was introduced in python 3.3